src/App.js

```diff
@@ -1,7 +1,46 @@
 import "./App.css";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Header from "./components/Header.js";
+import Nav from "./components/Nav.js";
+import Characters from "./components/Characters.js";
+import CharactersId from "./components/CharactersId.js";
+import Home from "./components/Home.js";
+import Footer from "./components/Footer.js";
 
 function App() {
-  return <div className="App">Rick & Morty</div>;
+  return (
+    <Router>
+      {" "}
+      <div className="App">
+        <header className="Header-Grid">
+          <div className="Top-Menu">
+            <Header />
+            <Nav />
+          </div>
+        </header>
+        <main className="Main-Grid">
+          <Switch>
+            <Route path="/characters/:id">
+              <CharactersId />
```

Comment:
in line 23: is this colon needed, I am not sure. Are you using useParam? If not this wont work, i think. I am checking some resources. Let me now, if I am on wrong track.

Reply:
I am on the wrong track.

Reply:
I tried to follow Namir's example. Didn't include them at first